docs/docs/concepts/workflows.rst

```diff
@@ -2,34 +2,220 @@ Workflows
+- Easy tracking of configuration changes history, allowing for a quick overview of configuration evolution.
+
+Workflows in Klaw are managed by defining ownerships, roles, permissions, and authorizations. In the following section, you will find an overview of the key concepts related to ownership in Klaw.
 
 .. glossary::
 
     Request and approval
-      One of the biggest benefits of working with Klaw is that it adds governance to your Apache Kafka® landscape as it grows.  When a new topic, schema, ACL or connector is needed, developers **request** that item themselves.  Another member of the same team can then **approve** the request, and the item will be created.
+      One of the primary advantages of using Klaw is the governance it adds to the growing Apache Kafka® landscape. Developers can **request** new topics, schemas, ACLs, or connectors themselves. The request is then reviewed and **approved** by another member of the same team. 
+
         .. note::
-            Only holders of the ``USER`` role can make requests; users with the ``SUPERADMIN`` role cannot make requests, but can manage users and teams. This is the default configuration.
+            This is the default configuration. The default configuration permits only users with the  ``USER`` role to make requests. Users with the ``SUPERADMIN`` role cannot request but can manage users and teams.
+
+.. note::
+   A user who raises a request cannot approve it. Instead, a different user must approve.
+
+Approvals
+---------
+
+In Klaw, the approval process plays an essential role in managing workflows. This process entails verifying and accepting proposed changes, making it a crucial step toward enhancing the security and governance of the system.
+
+Different Roles
+````````````````
+
+Within this approval process, different roles come into play, each with its unique responsibilities and permissions. These roles include:
+
+#. **Resource Owners Team** : When transferring ownership between teams, the current resource owner provides the necessary approval.
+#. **Requestors Team** : Usually, a fellow team member reviews and either approves or declines a request.
+#. **All Approvers** : Users assigned the ``APPROVE_ALL_REQUESTS_TEAMS`` permission can approve any request from any team.
+#. **Admin** : Admin users manage Klaw, primarily reviewing and approving user requests unless they also assinged the ``APPROVE_ALL_REQUESTS_TEAMS`` permission.
+
+User Requests
+#############
+
+The matrix below provides an understanding of the roles that approve a user registration request.
+
+.. list-table:: New User Approver matrix
+   :header-rows: 1
+   :class: no-scroll
+
+   * - Operation Type
+     - Resource Owners Team
+     - Requestors Team
+     - All Approvers
+     - Admin
+   * - User Registration Request
+     -
+     -
+     -
+     - ✅
+
+Topic Requests
+##############
+
+Topic Owner
```

Comment:
Should "Topic Owner" be styled as a sub-heading? Same for the other Requests below.